mathesar/api/db/viewsets/columns.py

```diff
@@ -92,7 +92,9 @@ def create(self, request, table_pk=None):
             except InvalidTypeError as e:
                 raise database_api_exceptions.InvalidTypeCastAPIException(
                     e,
-                    message='This type casting is invalid.',
+                    message=f'{e.column_name} cannot be casted to {e.new_type}.'
```

Comment:
Also, it would be much clearer to construct the message outside of the call. You could either
- add a method to the exception to construct the message (since it's always the same), or
- put a method somewhere else convenient.

Reply:
Ok, I'll refactor that.